ssl/statem/statem_lib.c

```diff
@@ -2365,23 +2365,23 @@ int ssl_choose_client_version(SSL_CONNECTION *s, int version,
         real_max = ver_max;
 
     /* Check for downgrades */
-    if (s->version == TLS1_2_VERSION && real_max > s->version) {
-        if (memcmp(tls12downgrade,
+    if (!SSL_CONNECTION_IS_DTLS(s) && real_max > s->version) {
```

Comment:
I suggest adding a comment here as a hint for us to review this code when DTLSv1.3 goes in:

`/* TODO(DTLSv1.3): Update this code for DTLSv1.3 */`